nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-resources/src/main/resources/conf/bootstrap.conf

```diff
@@ -66,6 +66,20 @@ nifi.bootstrap.protection.hashicorp.vault.conf=./conf/bootstrap-hashicorp-vault.
 # AWS KMS Sensitive Property Providers
 nifi.bootstrap.protection.aws.kms.conf=./conf/bootstrap-aws.conf
 
+# Note: the following mapping properties only apply if a Sensitive Property Provider that uses property contexts
+# is configured.  Otherwise, these values are ignored.
+#
+# If no nifi.bootstrap.protection.context.mapping.* properties are provided, the context for protected
+# properties uses a 'default' context, as in "default/Manager Password".  Properties in nifi.properties are always
+# assigned this context, but there is a possibility of naming conflicts among the .xml configuration files.
+#
+# To create separate contexts for XML properties, you may provide context mappings in the format:
```

Comment:
Although the most common use case relates to XML-based configuration files, recommend rewording to avoid referencing the storage format since it shouldn't be specific to the implementation.